### DISCUSSION TITLE
How to quantify the spatial distribution of the induced field?

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#246] [9. Results and analysis]
Hi SimNIBS team, thanks for the great tool!
Is there any metric to calculate and quantify the spatial distribution of physical fields from simNIBS simulations? For example, how do I measure optimal focusing when optimizing?
Thanks!

**Comment by TorgeW** [comment]
Have a look at these two papers, they describe possible metrics which you can implement based on the results of the electric field simulations.

https://backend.orbit.dtu.dk/ws/portalfiles/portal/245616065/2020.05.27.118422.full.pdf
https://research.ku.dk/search/result/?pure=en/publications/accessibility-of-cortical-regions-to-focal-tes(f939d24c-eb4c-47aa-8d4b-0b222d202b19).html
